src/bnb_trading/optimal_levels.py

```
@@ -218,213 +220,247 @@
             >>> analyzer = OptimalLevelsAnalyzer(config)
         """
         self.price_interval = 25  # Интервал между ценови нива
-        self.min_touches = 3      # Минимум докосвания за валидно ниво
-        
+        self.min_touches = 3  # Минимум докосвания за валидно ниво
+
         logger.info("Optimal Levels анализатор инициализиран")
-    
+
     def analyze_optimal_levels(self, daily_df: pd.DataFrame, weekly_df: pd.DataFrame) -> Dict:
```

Ruff (0.12.2)

Unused method argument: `daily_df`

(ARG002)